{
"version":3,
"file":"module$node_modules$rxjs$dist$cjs$internal$util$ArgumentOutOfRangeError.js",
"lineCount":1,
"mappings":"AAAAA,cAAA,wEAAA,CAA4F,QAAQ,CAACC,MAAD,CAAQC,OAAR,CAAgBC,MAAhB,CAAuBC,OAAvB,CAAgC,CAEpIC,MAAAC,eAAA,CAAsBF,OAAtB,CAA+B,YAA/B,CAA6C,CAAEG,MAAO,CAAA,CAAT,CAA7C,CACAH,QAAAI,wBAAA,CAAkC,IAAK,EACnCC,OAAAA,CAAqBP,OAAA,CAAQ,kEAAR,CACzBE,QAAAI,wBAAA,CAAkCC,MAAAC,iBAAA,CAAoC,QAAS,CAACC,MAAD,CAAS,CACpF,MAAOH,SAAgC,EAAG,CACtCG,MAAA,CAAO,IAAP,CACA,KAAAC,QAAA,CAAe,uBAFuB,CAD0C,CAAtD,CALkG;",
"sources":["node_modules/rxjs/dist/cjs/internal/util/ArgumentOutOfRangeError.js"],
"sourcesContent":["shadow$provide[\"module$node_modules$rxjs$dist$cjs$internal$util$ArgumentOutOfRangeError\"] = function(global,require,module,exports) {\n\"use strict\";\nObject.defineProperty(exports, \"__esModule\", { value: true });\nexports.ArgumentOutOfRangeError = void 0;\nvar createErrorClass_1 = require(\"./createErrorClass\");\nexports.ArgumentOutOfRangeError = createErrorClass_1.createErrorClass(function (_super) {\n    return function ArgumentOutOfRangeError() {\n        _super(this);\n        this.message = 'argument out of range';\n    };\n});\n//# sourceMappingURL=ArgumentOutOfRangeError.js.map\n};"],
"names":["shadow$provide","global","require","module","exports","Object","defineProperty","value","ArgumentOutOfRangeError","createErrorClass_1","createErrorClass","_super","message"]
}
